### ISSUE TITLE
HTTP Request Body (`Other` type) keeps changing back to previous content after click "Send"

### ISSUE THREAD

**Issue by kumanoko24** [getinsomnia/insomnia#6286]
### Expected Behavior

The request should be sent with new content of request body

### Actual Behavior

The request is being sent with previous content request body, and the request body content also changes back to previous one

### Reproduction Steps

1. Update Insomnia from version 2023.4.x to 2023.5.1
2. Go to any HTTP POST request
3. Use 'Other' body type
4. Change the content of the request body
5. Click Send
6. See the response and request body 

### Is there an existing issue for this?

- [x] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

Insomnia info:
```
Version: Insomnia 2023.5.1
Build date: 8/15/2023
OS: Darwin arm64 21.6.0
Electron: 25.2.0
Node: 18.15.0
Node ABI: 116
V8: 11.4.183.23-electron.0
Architecture: arm64
```

### Insomnia Version

2023.5.1

### What operating system are you using?

macOS

### Operating System Version

macOS Monterey 12.6.7

### Installation method

Insomnia.Core-2023.5.1.dmg

### Last Known Working Insomnia version

2023.4.0

_No response_

**Comment by kumanoko24** [comment]
After downgrade to :

```
Version: Insomnia 2023.4.0
Build date: 7/4/2023
OS: Darwin arm64 21.6.0
Electron: 23.3.9
Node: 18.12.1
V8: 11.0.226.20-electron.0
Architecture: arm64
```

This behaviour does not happen again.

**Comment by jackkav** [comment]
Any other information you can give us to reproduce this would be useful as it doesn't happen for us.

**Comment by kumanoko24** [comment]
Hi @jackkav , thanks for your prompt reply.

I will try reproducing with 2023.5.1 asap and check detailed logs, since I need insomnia working for a moment.

**Comment by sulidev** [comment]
Hi there! I'm experiencing this issue too. For me it is the 'Other' body type that is buggy. Changing that to 'JSON' fixed it.

**Comment by kumanoko24** [comment]
@sulidev @jackkav Ah, I must be too busy and hurried yesterday to notice this, yes I used the 'Other' body type and got this situation.

Because I was using Insomnia to interact with SurrealDB, the only way was to use 'Other' body type.

I will update the issue report content.

**Comment by kumanoko24** [comment]
Reproduction Steps updated

**Comment by jackkav** [comment]
Could you please confirm if this is fixed in 2023.5.4
